src/containers/thankyoupage/ThankYouPage.js

```diff
@@ -24,6 +24,9 @@ const ThankYouPage = (props) => {
           <a href="https://www.instagram.com/FloNightingaleRN/">
             <i className="fa fa-instagram"></i>
           </a>
+          <a href="mailto:floswhistle@gmail.com?subject=Flo's Whistle: Pandemic">
+            <i className="far fa-envelope"></i>
```

Comment:
Let's add a span with the text of the email address as well for accessibility/people who hate their mail client and want to copy paste

Reply:
Will add this in with the next PR for the download page. I have the page built, it just needs the csv download functionality now.